zerver/webhooks/slack/view.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+from django.utils.translation import ugettext as _
+from django.http import HttpRequest, HttpResponse
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_message, create_stream_if_needed
+from zerver.lib.response import json_success, json_error
+from zerver.lib.validator import check_string, check_int
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.models import UserProfile, Client
+
+ZULIP_MESSAGE_TEMPLATE = "**{message_sender}** has sent the following message:\n `{text}`"
+
+@api_key_only_webhook_view('Slack')
+@has_request_variables
+def api_slack_webhook(request, user_profile, client,
+                      user_name=REQ(),
+                      text=REQ(),
+                      channel_name=REQ(),
+                      stream=REQ(default='slack'),
+                      channels_map_to_topics=REQ(default=1)):
+    # type: (HttpRequest, UserProfile, Client, str, str, str, str, int) -> HttpResponse
+
+    if int(channels_map_to_topics):
```

Comment:
We still don't have any kind of validation here.
Please run a script when `channels_map_to_topics` has value `abc` -> ERROR